CONTRIBUTING.md

```diff
@@ -578,7 +578,9 @@ See also:
 The tests should never leak goroutines.
 
 Use the term `ConcurrentSafe` in the test name when it aims to verify the
-absence of race conditions.
+absence of race conditions. The top-level tests with this term will be run
+100 times in the `test-concurrent-safe` CI job to increase the chance of
```

Comment:
```suggestion
many times in the `test-concurrent-safe` CI job to increase the chance of
```

This will be easier to maintain if we decide to change the test run count 😉